### PR TITLE
Ria 3758 1

### PR DESCRIPTION
**Pull request by KumarAkepati**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/features/add-case-note.feature [hunk 1]
```diff
@@ -45,24 +50,22 @@ Feature: Add a case note
     And within the `Case notes` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date added` field
 
     When I click the `Add case note` link
+    And I wait for 3 seconds
     Then I am on the `Add case note` page
 
 
-  @regression @add-case-note @RIA-575
+  @regression @add-case-note @RIA-575 @nightly-test1234
```
Comment (Srihari-Nakkala) [inline e2e/features/add-case-note.feature:57]: Please check the Tag

Comment (KumarAkepati) [reply e2e/features/add-case-note.feature]: Done

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

File: e2e/features/end-appeal.feature [hunk 2]
```diff
@@ -42,39 +50,39 @@ Feature: End appeal
     And I should not see the hearing details
     And I should see the case details
 
-    When I switch to be a `Judge`
```
[the remark this answers is not in the document]
Comment (KumarAkepati) [reply e2e/features/end-appeal.feature:45]: done

File: e2e/features/flag-case.feature [hunk 3]
```diff
@@ -113,7 +113,7 @@ Feature: Flag a case
     And I should see `Oh no! This case turned out to be rather complex!` for the `Additional information` field
     And I should see the text `The person has once slammed their fists against the desk.`
 
-  @regression @flag-case @RIA-1742 
+  @regression @flag-case @RIA-1742 @nightly-test1
```
[the remark this answers is not in the document]
Comment (KumarAkepati) [reply e2e/features/flag-case.feature]: done

File: e2e/features/flag-case.feature [hunk 4]
```diff
@@ -151,7 +151,7 @@ Feature: Flag a case
     And I should see `Oh no! This case turned out to be rather complex!` for the `Additional information` field
     And I should not see the text `The person has once slammed their fists against the desk.`
 
-  @regression @flag-case @RIA-3865
+  @regression @flag-case @RIA-3865 @nightly-test1
```
[the remark this answers is not in the document]
Comment (KumarAkepati) [reply e2e/features/flag-case.feature]: done

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Pull request merged**